cmd/temporalite/main.go

```diff
@@ -162,6 +163,13 @@ func buildCLI() *cli.App {
 					return cli.Exit(fmt.Sprintf("ERROR: only one of %q or %q flags may be passed at a time", ephemeralFlag, dbPathFlag), 1)
 				}
 
+				// Make sure the default db path exists (user does not specify path explicitly)
+				if !c.IsSet(dbPathFlag) {
```

Comment:
Can you confirm this is not considered "set" when left as default?

Reply:
Yes this leads to the correct behavior. I'd like to add an integration test though.